lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -169,28 +169,32 @@ func (c *ContainerProxy) Client() *client.PortLayer {
 //
 // returns:
 //	(containerID, containerHandle, error)
-func (c *ContainerProxy) CreateContainerHandle(imageID string, config types.ContainerCreateConfig) (string, string, error) {
-	defer trace.End(trace.Begin(imageID))
+func (c *ContainerProxy) CreateContainerHandle(vc *viccontainer.VicContainer, config types.ContainerCreateConfig) (string, string, error) {
+	defer trace.End(trace.Begin(vc.ImageID))
 
 	if c.client == nil {
 		return "", "", InternalServerError("ContainerProxy.CreateContainerHandle failed to create a portlayer client")
 	}
 
-	if imageID == "" {
+	if vc.ImageID == "" {
```

Comment:
Just tested with a v0.8.0 container - inspect works fine, although the problem addressed by this PR still exists. There's nothing we can do about that, though, but it isn't an issue since we didn't support `compose` in v0.8 anyway.